prime-router/metadata/fhir_mapping/hl7/datatype/Identifier_Observation.yml

```diff
@@ -0,0 +1,27 @@
+#
+# (C) Copyright IBM Corp. 2021
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Creates an identifier for observation which concatenates value strings
+---
+system:
+  type: SYSTEM_URL
+  valueOf: $sys
+  expressionType: HL7Spec
+
+value_1:
```

Comment:
These files are hard to review when we move them from the library.  Add a couple of comments here on what you changed to make it easier for reviewers to find your changes.